pipelines/Jenkinsfile

```
@@ -0,0 +1,248 @@
+              cd com.etendoerp.docker
+              git checkout ${GIT_COMMIT}
+              cd ../..
+              ./gradlew prepareConfig
+              ./gradlew setup
+              ./gradlew resources.up -Pcontext.name=${CONTEXT_NAME}
+              """
+
+              echo "---------------- Verifying Docker Containers ----------------"
+              def dockerPsOutput = sh(script: 'docker ps', returnStdout: true)
+              if (!dockerPsOutput.contains("etendo-db-1")) {
+                error("The etendo-db-1 container is not running")
+              }
+
+              echo "---------------- Docker Containers are running ----------------"
+              
+              sh """
+              cd etendo_base
+              ./gradlew resources.stop -Pcontext.name=${CONTEXT_NAME}
+              """
+              
+              echo "---------------- Cleaning Docker Containers ----------------"
+              sh """
+              cd etendo_base
+              ./gradlew resources.down -Pcontext.name=${CONTEXT_NAME}
+              """
+              
+              dockerPsOutput = sh(script: 'docker ps', returnStdout: true)
+              if (dockerPsOutput.contains("etendo-db-1")) {
+                error("The etendo-db-1 container still running")
+              }
+
+              currentBuild.result = SUCCESS
+            } catch (Exception e) {
+              echo "---------------- Build environment Failed ----------------"
+              echo "Exception occurred: " + e.toString()
+              currentBuild.result = FAILED
+              error('Build Failed')
+            }
+          }
+        }
+      }
+    }
+  }
+  
+  post {
+    success {
+      container('compiler') {
+        script {
+          sh './utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker success "Successful Bundles Test" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
+        }
+      }
+    }
+
+    fixed {
+      container('compiler') {
+        script {
+          sh './utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker success "Successful Bundles Test" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
+        }
+      }
+      mail to: EMAIL_ADDRESS,
+      subject: "✅ FIXED - ${currentBuild.fullDisplayName}",
+      body: """
+${new Date()}
+__________________________________________________________
+
+✅ ERRORS FIXED ✅
+
+This is a compatibility test of etendo_base execution with all installed bundles.
+
+The problems found in the previous run/s have been fixed!  💪
+Check out the output in the following link: 
+${env.BUILD_URL}
+__________________________________________________________
+        """
+    }
+    failure {
+      container('compiler') {
+        script {
+          sh './utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker failure "Build Failed" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
```

@RomanMagnoli save the package in an environment variable
`MODULE_PACKAGE = "com.etendoerp.docker"`

```suggestion
          sh './utils/repo-update-scripts/build-update-github.sh $MODULE_PACKAGE failure "Build Failed" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
```

---

@RomanMagnoli 
Add 

envinronment {
.....
    COMMIT_INPROGRESS_STATUS = "pending"
    COMMIT_SUCCESS_STATUS    = "success"
    COMMIT_FAILED_STATUS     = "failure"
}

```suggestion
          sh './utils/repo-update-scripts/build-update-github.sh com.etendoerp.docker $COMMIT_FAILED_STATUS "Build Failed" $ACCESS_TOKEN $CORE_GIT_COMMIT $BUILD_URL \"${CONTEXT_BUILD}\"'
```

---

Fixed!